music_assistant/helpers/audio.py

```diff
@@ -836,6 +837,13 @@ def get_player_filter_params(
 
     dsp = mass.config.get_player_dsp_config(player_id)
 
+    if player := mass.players.get(player_id):
+        is_grouped = bool(player.synced_to) or bool(player.group_childs)
+        if is_grouped and PlayerFeature.MULTI_DEVICE_DSP not in player.supported_features:
+            # We can not correctly apply DSP to a grouped player without multi-device DSP support,
+            # so we disable it.
+            dsp.enabled = False
```

Comment:
If you are wondering about a way how e can tell the user what is happening:

I was thinking that we should maybe also extend streamdetails the same way we do now for volume normalization etc. That way the frontend can show the user exactly the whole chain:

- The input file/stream/codec
- Filters like volume normalization
- DSP on/off and what filters
- Output path (incl lossy or lossless flag and sample rate etc)

Related to this one here: https://github.com/music-assistant/frontend/pull/651